polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -367,6 +367,18 @@ mlir::Attribute MLIRScanner::InitializeValueByInitListExpr(mlir::Value ToInit,
       return mlir::DenseElementsAttr();
     }
 
+    if (auto *AIL = dyn_cast<ArrayInitLoopExpr>(Expr)) {
+      // In-house bitcast from `memref<-xarray<N x Ty>>` to `memref<? x Ty>`
```

Comment:
Nit: `-` in `memref<..>` (or does that have a special meaning?)

Reply:
I meant it could be `?` or a `number`

Reply:
I think it's fine then, bummer that `memref` already uses all the good placeholder chars 🤔

Reply:
Yes, they left us no good choice...